README.md

```diff
@@ -53,7 +52,21 @@ class ExpensiveRenderer extends React.Component {
 }
 ```
 
-## Benchmarking
+## Do I Need `shouldComponentUpdate`?
+
+> What's faster than a really fast deep comparion? No deep comparison at all.
+
+—This Readme
+
+
+Deep checks in React's `shouldComponentUpdate` should not be used blindly.
+First, see if a
+[PureComponent](https://reactjs.org/docs/react-api.html#reactpurecomponent) would
+work for you. If it won't (if you need deep checks), make sure you've correctly indentified the bottleneck in
```

Comment:
Suggestion: `If it won't (because you need deep checks), a good next step is to make sure`